sdks/java/io/tika/src/test/java/org/apache/beam/sdk/io/tika/TikaIOTest.java

```diff
@@ -18,246 +18,156 @@
-    }
+    p.apply("ParseInvalidPdfFile", FileIO.match().filepattern(resourcePath))
+      .apply(FileIO.readMatches())
+      .apply(TikaIO.parseAll());
+    p.run();
   }
 
   @Test
   public void testReadDisplayData() {
-    TikaIO.Read read = TikaIO.read()
-        .from("foo.*")
+    TikaIO.ParseAll read = TikaIO.parseAll()
         .withTikaConfigPath("tikaconfigpath")
-        .withContentTypeHint("application/pdf")
-        .withMinimumTextlength(100)
-        .withReadOutputMetadata(true);
+        .withContentTypeHint("application/pdf");
 
     DisplayData displayData = DisplayData.from(read);
 
-    assertThat(displayData, hasDisplayItem("filePattern", "foo.*"));
     assertThat(displayData, hasDisplayItem("tikaConfigPath", "tikaconfigpath"));
     assertThat(displayData, hasDisplayItem("inputMetadata",
-        "[Content-Type=application/pdf]"));
-    assertThat(displayData, hasDisplayItem("readOutputMetadata", "true"));
-    assertThat(displayData, hasDisplayItem("parseMode", "asynchronous"));
-    assertThat(displayData, hasDisplayItem("queuePollTime", "50"));
-    assertThat(displayData, hasDisplayItem("queueMaxPollTime", "3000"));
-    assertThat(displayData, hasDisplayItem("minTextLen", "100"));
-    assertEquals(8, displayData.items().size());
-  }
-
-  @Test
-  public void testReadDisplayDataSyncMode() {
-    TikaIO.Read read = TikaIO.read()
-        .from("foo.*")
-        .withParseSynchronously(true);
-
-    DisplayData displayData = DisplayData.from(read);
-
-    assertThat(displayData, hasDisplayItem("filePattern", "foo.*"));
-    assertThat(displayData, hasDisplayItem("parseMode", "synchronous"));
+        "Content-Type=application/pdf"));
     assertEquals(2, displayData.items().size());
   }
 
   @Test
-  public void testReadDisplayDataWithDefaultOptions() {
-    final String[] args = new String[]{"--input=/input/tika.pdf"};
-    TikaIO.Read read = TikaIO.read().withOptions(createOptions(args));
-
-    DisplayData displayData = DisplayData.from(read);
-
-    assertThat(displayData, hasDisplayItem("filePattern", "/input/tika.pdf"));
-    assertThat(displayData, hasDisplayItem("parseMode", "asynchronous"));
-    assertThat(displayData, hasDisplayItem("queuePollTime", "50"));
-    assertThat(displayData, hasDisplayItem("queueMaxPollTime", "3000"));
-    assertEquals(4, displayData.items().size());
-  }
-  @Test
   public void testReadDisplayDataWithCustomOptions() {
-    final String[] args = new String[]{"--input=/input/tika.pdf",
-                                       "--tikaConfigPath=/tikaConfigPath",
-                                       "--queuePollTime=10",
-                                       "--queueMaxPollTime=1000",
-                                       "--contentTypeHint=application/pdf",
-                                       "--readOutputMetadata=true"};
-    TikaIO.Read read = TikaIO.read().withOptions(createOptions(args));
+    TikaIO.ParseAll read = TikaIO.parseAll()
```

Comment:
Rename "read" to "parseAll" in variables and test method names throughout this test?